### DISCUSSION TITLE
vuecli项目 静态资源路径补全出错

### DISCUSSION THREAD

**Discussion by 2392232844qq** [jd-opensource/micro-app#738] [Q&A]
按照官方文档配置了
if (window.__MICRO_APP_ENVIRONMENT__) {
  // eslint-disable-next-line
  __webpack_public_path__ = window.__MICRO_APP_PUBLIC_PATH__
}
子应用的静态资源路径还是 基座的前缀,请问这个改怎么解决

**Comment by bailicangdu** [comment]
代码是对的，检查一下文件名和文件引入的地方对吗

**Reply by 2392232844qq** [reply]
是需要一些其他的配置吗

**Reply by 2392232844qq** [reply]
文件名和引入是没有问题 也成功设置了 我打印了信息
webpack_public_path 这个成功被设置为子应用的域名了 但是加载的图片还是基座应用的域名

**Reply by bailicangdu** [reply]
![image](https://user-images.githubusercontent.com/20297227/226532325-332bd097-b64a-4015-8d97-60269b61528f.png)
这3个地方都对吗

**Reply by 2392232844qq** [reply]
都是对的

**Reply by bailicangdu** [reply]
那就不应该了，能提供一个demo看一下吗

**Reply by 2392232844qq** [reply]
我把子应用的文件发你可以吗

**Reply by 2392232844qq** [reply]
可以加个其他联系方式吗

**Reply by bailicangdu** [reply]
可以先加一下微信群 https://github.com/micro-zoe/micro-app/blob/dev/Contact.md

**Reply by 2392232844qq** [reply]
好的

**Reply by 2392232844qq** [reply]
这个二维码过期了

**Reply by 2392232844qq** [reply]
重新提供一个吧

**Reply by bailicangdu** [reply]
已更新

**Comment by zjluse** [comment]
hi, 我的是vite， 遇到同样的问题，请问是配置问题吗？css里面的背景图片补全访问到了主应用的路径。麻烦看下
